kythe/go/serving/pipeline/beamio/BUILD

```diff
@@ -0,0 +1,29 @@
+load("//tools:build_rules/shims.bzl", "go_library", "go_test")
+
+package(default_visibility = ["//kythe:default_visibility"])
+
+go_library(
+    name = "beamio",
+    srcs = glob(
```

Comment:
There's only one! Please let's just list it.

Reply:
✓